htdocs/master.inc.php

```diff
@@ -233,7 +233,14 @@ function is_countable($c)
 		// This is very problematic during a fiscal control.
 		$conf->global->INVOICE_DISABLE_REPLACEMENT = 1;
 	}
-
+	if ($mysoc->country_code == 'GR' && !isset($conf->global->INVOICE_DISABLE_REPLACEMENT)) {
+		// The replacement invoice type is not allowed in Greece.
+		$conf->global->INVOICE_DISABLE_REPLACEMENT = 1;
+	}
+	if ($mysoc->country_code == 'GR' && !isset($conf->global->INVOICE_DISABLE_DEPOSIT)) {
+		// The deposit invoice type is not allowed in Greece.
+		$conf->global->INVOICE_DISABLE_DEPOSIT = 1;
```

Comment:
The replacement invoice is not allowed into a lof of country.
However it is more strange for deposit invoice (also known as down payment).
When a company receive a down payment, before the final amount is know. What is the process in greece. No invoice is generated ?

Reply:
For down payments(advance) there is no legal obligation to issue an invoice but instead for cash payments only we issue a simple receipt referencing the client. In bank payments the bank statement is considered as receipt because payment is made to a "professional account".

For more context please read this [issue](https://github.com/Dolibarr/dolibarr/issues/23076) 

So in case of an audit things get messy

Since then the solution we've came up with to make dolibarr compliant with Greek legislation is by using the following modules:
* https://www.dolistore.com/en/modules/1834-CTPT.html    
it creates the mask using {tttt} 

* https://github.com/evansgl/dolibarr-mydata                        
it uploads invoice data to ministry database

* https://github.com/TechnickSysAid/mask4greece              
it generates pdf retrieving data from mydata module plus other requirements 

Any suggestion or insight as to how we may proceed would be greatly appreciated
Thank you